documentation/aircraft-commands.md

```diff
@@ -142,9 +141,7 @@ _Syntax -_ `AAL123 dvs` or `AAL123 dvs [alt]`
 
 ### ILS
 
-_Aliases -_ `ils` / `i`
-
-_Shortkey -_ `numpad *`
+_Aliases -_ `ils` / `i` / `*`
```

Comment:
WIth this and other cases where shortkeys/hotkeys are being moved to an alias, we're losing context on some purposeful functionality. The shortkey/hotkey (same meaning iirc, just not all using the same word) implies that you can press _that key_ to insert the command _with the appropriate spaces surrounding it_.

So in this case, while from the parser's point of view, `*` is simply an alias for `ils`, from the user's point of view they have the following options to utilize this command:
- type `ils` followed by a space and the runway number
- type `i` followed by a space and the runway number
- press the `*` key on the keyboard, followed by the runway number (with no space separating them)

The hotkeys also have the advantage of being able to be chained without pressing the space bar. For example, the following key strokes `[right arrow] 2 5 0 [down arrow] 3 0 * 2 8 R` would yield `t r 250 d 30 * 28R` in the command bar without ever touching the space key.

So somehow, we do need to differentiate by which keys perform this behavior and which are simply characters that need to be entered in the command bar for it to be understood by the parser.

Reply:
Ah, right. It's worth highlighting that behavior of *numpad*  `+`, `-` and `*`.

On the other hand I'd wanted to make it explicit that inputting them using non-numpad keys will also be acceptable to the parser -- in contrast with, say, the arrow keys, which when pressed are substituted by the corresponding actual commands.

So... both?